ci/util.py

```diff
@@ -322,6 +322,12 @@ def _running_on_ci():
     return 'CC_ROOT_DIR' in os.environ
 
 
+def current_config_set():
```

Comment:
+ `_name` - or return the cfg set (from cfg_factory)?

Reply:
done